pipeline/merger/src/test/scala/weco/pipeline/merger/services/TeiOnMergerScenarioTest.scala

```diff
@@ -127,17 +154,72 @@ class TeiOnMergerScenarioTest
       .getMerged(secondInternalWork)
       .data
       .items should contain allElementsOf physicalSierra.data.items
+
+    And("the tei internal works retain their collectionsPath")
```

Comment:
We should assert that the top-level work also retains its collection path, so we know we can build the collection tree.

Reply:
This is a bit of a weird test because it's a case that shouldn't be happening where internal works have collectionspath but the wrapper work doesn't. It should log a warning at least cause it's a data issue and probably a transformer bug if it happens, so I'll add that. But it needed to be tested because it has to be dealt with in the match statement. [this test](https://github.com/wellcomecollection/catalogue-pipeline/pull/1905/files#diff-0b198acf186857fb7019d066fba330d14bab1b0e0b965fd4ad1106b437f0befbR192) tests the case where the tei work is not merged and the collectionpath are correct and [this other test](https://github.com/wellcomecollection/catalogue-pipeline/pull/1905/files#diff-0b198acf186857fb7019d066fba330d14bab1b0e0b965fd4ad1106b437f0befbR58) tests the case where a calm work is redirected to the tei work and so the fake collectionpath is blatted